### ISSUE TITLE
There is no tooltip that displays the full websire URL from a container

### ISSUE THREAD

**Issue by Softvision-VladBacia** [mozilla/testpilot-containers#615]
**[Affected versions]:**
- Firefox 54
- Containers Experiment Build: 2.3.0

**[Affected Platforms]:**
- All Windows
- All Linux
- All Mac

**[Prerequisites]:**
- Containers Experiment is installed on a new browser profile.

**[Steps to reproduce]:**
1. Start the browser with the profile from prerequisites.
2. Hover the "+" open new tab button and open a New Container tab.
3. In the container tab access a webpage (www.mozilla.com)
4. Click on the Containers button from the browser toolbar.
5. Open the details page from the Containers doorhanger for the used container.
6. Hover the website name that is displayed.

**[Expected results]:**
- Full container name is displayed in a tooltip.

**[Actual results]:**
- There is no tooltip with the name and URL of the webpage.

**[Notes]:**
- Attached a screenshot of the issue:
![no page tooltip](https://user-images.githubusercontent.com/9248513/27393123-0ff104b4-56b1-11e7-8985-fef21abaa9c6.jpg)

**Comment by kjozwiak** [comment]
We've talked about adding several tooltips throughout the containers test pilot in #112. It would definitely be nice if users could see the entire URL, but I don't think this blocker for the Shield Study release.

I renamed #112 and added this bug [here](https://github.com/mozilla/testpilot-containers/issues/112#issuecomment-310241334) as another component/area that is in need of a tooltip.

If we're fine tackling this in #112, we can just close this off.
